README.md

```diff
@@ -67,7 +67,8 @@ the application config.
   config :formulator,
     translate_error_module: YourAppName.ErrorHelpers,
     validate: false, # defaults to true
-    validate_regex: false  # defaults to true
+    validate_regex: false,  # defaults to true
+    wrapping_class: "input-set"  # defaults to nil
```

Comment:
Is it more common to be called a `wrapper_class` ?

Reply:
I have no opinion: @creuter, any preference?

Reply:
Same, I don't have strong opinions.

`wrapper` is the thing, `wrapping` is what the thing is doing. We usually name things nouns. Let's go with `wrapper_class`

Here's how simple form uses it:   `<%= f.input :username, wrapper_html: { class: 'username' } %>`